index.css

```diff
@@ -1,13 +1,110 @@
+:root {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-size: 1.6rem;
+}
+
+
 .hero {
   background-image: url("./assets/01_Main_Hero_Mobile.jpg");
   background-size: cover;
   background-position: center 33%;
   height: 100vh;
   width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+header {
+  padding: 2rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+header .logo {
+  font-size: 2.5rem;
+  text-transform: uppercase;
+}
+
+.full-menu {
+  display: none;
+}
+
+header img {
+  width: 4rem;
+}
+
+.container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  align-items: center;
+
+}
+
+.svg {
+  flex: 3;
+}
+
+.svg img {
+  width: 40rem;
+
+}
+
+.content {
+  color: white;
+  font-weight: 800;
+  margin: auto;
+  text-transform: uppercase;
+  flex: 1;
+  width: 45rem;
+}
+
+.text {
+  text-align: center;
+  line-height: 2.5rem;
+  word-spacing: .2rem;
+}
+
+.button {
+  padding: 1.2rem;
+  border: 3px solid white;
+  border-bottom-right-radius: 2rem;
+  text-align: center;
+  width: 75%;
+  margin: 5rem auto;
 }
 
-@media (min-width: 700px) {
+
+@media (min-width: 960px) {
   .hero {
     background-image: url("./assets/01_Main_Hero_Desktop.jpg")
   }
+
+  .full-menu {
+    display: block;
+
+  }
+
+  ul {
```

Comment:
you should not add such specific styling to a generic tag.
Imagine you (or a team colleague) developing further and want to include a list again. If you want to use `<ul>` now, it differs from the standard behaviour and you have to change it in both places.
So it would be bether to add a custom class for this.